addons/mail/static/src/chatter/web/chatter.xml

```diff
@@ -42,9 +42,9 @@
                 </FileUploader>
                 <t t-else="" t-call="mail.Chatter.attachFiles"/>
                 <div class="o-mail-Followers d-flex me-1">
-                    <Dropdown position="'bottom-end'" menuClass="'o-mail-Followers-dropdown d-flex flex-column'" state="followerListDropdown">
-                        <button t-att-class="'o-mail-Followers-button btn btn-link d-flex align-items-center text-action px-1 ' + (props.compactHeight ? '' : 'my-2')" t-att-disabled="isDisabled" t-att-title="followerButtonLabel">
-                            <i class="fa fa-user-o me-1" role="img"/>
+                    <Dropdown position="'bottom-end'" menuClass="'o-mail-Followers-dropdown d-flex flex-column bg-100 border-secondary'" state="followerListDropdown">
+                        <button t-att-class="'o-mail-Followers-button btn btn-link d-flex align-items-center text-action px-1 ' + (props.compactHeight ? '' : 'my-2') + ' ' + (state.thread.selfFollower ? 'active' : '')" t-att-disabled="isDisabled" t-att-title="followerButtonLabel">
+                            <i class="fa fa-fw" role="img" t-att-class="{ 'fa-user': state.thread.selfFollower, 'fa-user-o': !state.thread.selfFollower }"/>
```

Comment:
Feels the right time to use `t-att-class` object syntax for `<button/>`, now that it has 2 different conditionals, for improved readability.